wordpress-mobile/WordPress-iOS
# [Gutenberg] Readable content margins

This PR makes the necessary changes to show properly the readable content margins implemented here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/769

![ReadableContent](https://user-images.githubusercontent.com/9772967/54772906-b31d4500-4c08-11e9-8d2a-c22fbab90f04.png)

To test:
- Run the project on an iPad.
- Be sure to have Metro running on the `issue/604-add-readable-margins` branch.
- Start an instance of Gutenberg.
- Make sure that the content is centered as shown in the screenshots.
- Run the project on an iPhone X or XS Max.
- On portrait, make sure that there are not side borders on selected blocks.
- On landscape, check that the content is centered and there are side borders on selected blocks.
